src/material-experimental/mdc-snack-bar/public-api.ts

```diff
@@ -16,7 +16,6 @@ export {
   MatSnackBarConfig,
   MatSnackBarDismiss,
   MatSnackBarRef,
-  SimpleSnackBar,
```

Comment:
I think this needs to be added back, still getting build failures about not finding `SimpleSnackBar`